docs/connector-development/config-based/tutorial/5-incremental-reads.md

```diff
@@ -11,8 +11,8 @@ First we'll update the spec block in `source_exchange_rates_tutorial/exchange_ra
 
 ```yaml
 spec: 
-  documentationUrl: https://docs.airbyte.io/integrations/sources/exchangeratesapi
-  connectionSpecification:
+  documentation_url: https://docs.airbyte.io/integrations/sources/exchangeratesapi
+  connection_specification:
```

Comment:
Why are you changing this? It doesn't look correct.

Reply:
I came accross this error by validating issue #20047 . Strictly following the tutorial example, both keywords in the yaml snippets seem to be incorrect. Changing them to the proposed naming resolves the error.

Reply:
This is issue has been stated in #18663

Reply:
Sorry I missed one update from low-code CDK https://github.com/airbytehq/airbyte/pull/18411/files#r1003903362